### ISSUE TITLE
Missing matching </div>

### ISSUE THREAD

**Issue by jofatmofn** [w3c/wcag#580]
https://www.w3.org/TR/WCAG20-TECHS/complete.html
Matching `</div>` for `<div class="technique">` missing for flash16 and flash30

**Comment by alastc** [comment]
Hi @jofatmofn, the old WCAG20-TECHS are being phased out, the new place is:
https://www.w3.org/WAI/WCAG21/Techniques/ 

I don't think the issue is there in the new pages.
